gcloud/bigquery/table.py

```diff
@@ -693,6 +699,225 @@ def insert_data(self,
 
         return errors
 
+    @staticmethod
```

Comment:
This is more a "learn test-fu from @tseaver than it is a review comment".

Why make it a `staticmethod`? The biggest piece of knowledge I feel I've gained from working with you is making **as much as possible** replace-able in tests. A `staticmethod` is more difficult to replace than a method on the module, so what's the use here?